publish.gradle

```diff
@@ -53,6 +60,7 @@ ext.configurePublish = { groupIdArg, artifactIdArg, descriptionArg ->
                         connection = libVcsUrl
                         developerConnection = libVcsUrl
                         url = libUrl
+                        tag = getGitHash()
```

Comment:
Oh, wow, that was much easier than I thought. :)

Reply:
I thought the same thing, lol